visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -48,9 +49,10 @@ export const cohortEditorStyles: () => IProcessedStyleSet<ICohortEditorStyles> =
     const theme = getTheme();
     return mergeStyleSets<ICohortEditorStyles>({
         cohortEditor: {
-            overflowY: 'visible',
+            overflowY: 'auto',
             width: '560px',
-            height: '624px',
+            height: '592px',
+            maxHeight: '592px !important',
```

Comment:
in general, using !important is never good. That is the case here, this is not good. Its used in this context because when not in its own layer, a scroll bar is added to the callout. On the whole, doNotLayer has its share of bugs.